src/desktop/apps/order2/__tests__/server.jest.js

```diff
@@ -0,0 +1,63 @@
+import { renderLayout } from "@artsy/stitch"
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { appHandler } from "../server"
+
+jest.mock("@artsy/stitch", () => ({
+  renderLayout: jest.fn(),
+}))
+
+jest.mock("reaction/Artsy/Router/server", () => ({
+  buildServerApp: jest.fn(),
+}))
+
+describe("Request for order", () => {
+  let req
+  let res
+  let next
+  const sendMock = jest.fn()
+
+  describe("/order2/:orderID", () => {
+    beforeEach(() => {
+      req = {
+        path: "/order2/123",
+        originalUrl: "/order2/123",
+        query: {},
+        header: () => "referrer",
+      }
+      res = {
+        locals: {
+          sd: {
+            CURRENT_USER: false,
```

Comment:
#minor but I think this will have a value of `undefined` or `null` when it doesn't exist, not `false`

Reply:
ugh, I was looking at a differend sd property when I wrote this!